core/modules/replica/AbortTransactionApp.cc

```diff
@@ -88,9 +106,54 @@ int AbortTransactionApp::runImpl() {
     job->start();
     job->wait();
 
-    // TODO: Analyze and display results
-
-    return 0;
+    if (_reportLevel > 0) {
+        
+        cout << "Job completion status: " << Job::state2string(job->extendedState()) << endl;
+
+        auto&& resultData = job->getResultData();
+        size_t numSucceeded = 0;
+        map<ExtendedCompletionStatus,size_t> numFailed;
+        resultData.iterate(
+            [&numFailed, &numSucceeded](SqlJobResult::Worker const& worker,
+                                        SqlJobResult::Scope const& table,
+                                        SqlResultSet::ResultSet const& resultSet) {
+                if (resultSet.extendedStatus == ExtendedCompletionStatus::EXT_STATUS_NONE) {
+                    numSucceeded++;
+                } else {
+                    if (numFailed.count(resultSet.extendedStatus) == 0) {
+                        numFailed[resultSet.extendedStatus] = 0;
+                    }
+                    numFailed[resultSet.extendedStatus]++;
+                }
+            }
+        );
+        cout << "Table processing summary:\n"
+             << "  succeeded: " << numSucceeded << "\n";
+        if (numFailed.size() == 0) {
+            cout << "  failed: " << 0 << endl;
+        } else {
+            cout << "  failed:\n";
+            for (auto&& itr: numFailed) {
```

Comment:
I understand it's `pair`, it was just my fantasy was falling short on how to name that "thing". So, I just kept calling them `itr`. Perhasp tis would work better:
```
for (auto&& entry: numFailed) {
    auto const extendedStatus = entry.first;
    auto const counter = entry.second;
    ...
```

Reply:
In C++17 you will be able to write it as:

    for (auto& [extendedStatus, counter]: numFailed) {
    }

and in Python you could do it since forever 😉 

    for extendedStatus, counter in numFailed.items():